src/test/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapperTest.java

```diff
@@ -80,11 +69,10 @@ class ScannerBootstrapperTest {
   @TempDir
   public Path tmpFolder;
 
-  private ScannerBootstrapper scannerBootstrapper;
+  private ScannerBootstrapper scannerBootstrapperSpy;
```

Comment:
I think that maintaining the name of the old field might be better: the spy wrapped around the type is an implementation detail that should not impact the other tests. If we need to document the fact that the field is a spy, I think we are better off leaving a comment on top of it